aten/src/ATen/native/LinearAlgebraUtils.h

```diff
@@ -213,62 +213,67 @@ static inline void squareCheckInputs(const Tensor& self) {
               "but they are ", self.size(-1), " by ", self.size(-2), " matrices");
 }
 
+/*
+ * Given a info int, obtained after a single operation, this function check if the computation
+ * has been successful (info = 0) or not, and report in case of the latter.
+ */
+static inline void singleCheckErrors(int64_t info, const char* name, int64_t batch_id=-1) {
+  std::stringstream batch_ss;
+  if (batch_id >= 0) {
+    batch_ss << ": For batch " << batch_id;
```

Comment:
```suggestion
    batch_ss << ": For batch element " << batch_id;
```

Reply:
Then, all the messages could start with capital letter (rather than with a ": " + lower case).